### PR TITLE
Remove utility class CompressedPropertyAccess

### PR DESCRIPTION
**Pull request by joakim-hove**
When https://github.com/OPM/opm-simulators/pull/2171 is merged the `CompressedPropertyAccess` class has no direct users.

As part of the ongoing 3D property refactoring this class would need a refactor: and since it is not in use at all I have opted to remove it.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2171 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2171 please

**Comment by joakim-hove** [comment]
jenkins build this please

File: CMakeLists_files.cmake [hunk 1]
```diff
@@ -110,8 +110,7 @@ if(HAVE_ECL_INPUT)
   list(APPEND TEST_SOURCE_FILES
 		tests/test_regionmapping.cpp
 		tests/test_ug.cpp
-		tests/test_compressedpropertyaccess.cpp
-                tests/cpgrid/grid_nnc.cpp
+    tests/cpgrid/grid_nnc.cpp
```
Comment (akva2) [inline CMakeLists_files.cmake]: meh.

Comment (joakim-hove) [reply CMakeLists_files.cmake]: Fixed

**Comment by joakim-hove** [comment]
objections? I'll merge in a couple of days

**Comment by atgeirr** [comment]
No objection, merge when green.

**Comment by atgeirr** [comment]
jenkins build this please

**Pull request merged**